dashboard/app/controllers/certificate_images_controller.rb

```diff
@@ -42,6 +42,9 @@ def show
   private
 
   def valid_course_name?(name)
-    name.nil? || name == ScriptConstants::ACCELERATED_NAME || CurriculumHelper.find_matching_course_version(name)
+    name.nil? ||
+      name == ScriptConstants::ACCELERATED_NAME ||
+      CertificateImage.prefilled_title_course?(name) ||
```

Comment:
I guess the source of truth for some of these IDs is technically the `cdo-tutorials` gsheet in Pegasus, but this seems more appropriate. :)